bin/build_olmsted_data.py

```diff
@@ -110,12 +111,21 @@ def pull_datasets(t):
    } 
 ]
 
-def create_node_records(tree):
+def create_node_records(tree, nt_seqs_dict, aa_seqs_dict, seqmeta_dict):
     records = []
     leaves_counter = 1
     for n in tree.traverse('postorder'):
         n.label = n.id = n.name
-        n.type = "node"
+	n.nt_seq = nt_seqs_dict[n.name]
```

Comment:
@eharkins Mmm... here too. Please convert all tabs to spaces if that's what's going on.

Reply:
Done